docs/content/Overlay.mdx

```diff
@@ -64,24 +56,212 @@ const Demo = () => {
+    type="React.RefObject<HTMLElement>"
+    description={
+      <>
+        Ref for the element to focus when the <InlineCode>Overlay</InlineCode> is closed.
+      </>
+    }
+  />
+  <PropsTableRow
+    required
+    name="onClickOutside"
+    type="function"
+    description={
+      <>
+        Function to call when clicking outside of the <InlineCode>Overlay</InlineCode>. Typically this function sets the{' '}
+        <InlineCode>Overlay</InlineCode> visibility state to <InlineCode>false</InlineCode>.
+      </>
+    }
+  />
+  <PropsTableRow
+    required
+    name="onEscape"
+    type="function"
+    description={
+      <>
+        Function to call when user presses <InlineCode>Escape</InlineCode>. Typically this function sets the{' '}
+        <InlineCode>Overlay</InlineCode> visibility state to <InlineCode>false</InlineCode>.
+      </>
+    }
+  />
+  <PropsTableRow
+    name="ignoreClickRefs"
+    type="React.RefObject<HTMLElement> []"
+    description={
+      <>
+        An array of ref objects to ignore clicks on in the onOutsideClick behavior. This is often used to ignore
+        clicking on the element that toggles the open/closed state for the Overlay to prevent the Overlay from being
+        toggled twice.
+      </>
+    }
+  />
+  <PropsTableRow
+    name="initialFocusRef"
+    type="React.RefObject<HTMLElement>"
+    description={
+      <>
+        Ref for the element to focus when the <InlineCode>Overlay</InlineCode> is opened. If nothing is provided, the
+        first focusable element in the <InlineCode>Overlay</InlineCode> body is focused.
+      </>
+    }
+  />
+  <PropsTableRow
+    name="width"
+    type={
+      <>
+        <InlineCode>small</InlineCode>, <InlineCode>medium</InlineCode>, <InlineCode>large</InlineCode>,{' '}
+        <InlineCode>xlarge</InlineCode>, <InlineCode>xxlarge</InlineCode>, <InlineCode>auto</InlineCode>
+      </>
+    }
+    defaultValue="auto"
+    description={
+      <>
+        Sets the width of the <InlineCode>Overlay</InlineCode>, pick from our set list of widths, or pass{' '}
+        <InlineCode>auto</InlineCode> to automatically set the width based on the content of the{' '}
+        <InlineCode>Overlay</InlineCode>. <InlineCode>small</InlineCode> corresponds to <InlineCode>256px</InlineCode>,{' '}
+        <InlineCode>medium</InlineCode> corresponds to <InlineCode>320px</InlineCode>, <InlineCode>large</InlineCode>{' '}
+        corresponds to <InlineCode>480px</InlineCode>, <InlineCode>xlarge</InlineCode> corresponds to{' '}
+        <InlineCode>640px</InlineCode>, <InlineCode>xxlarge</InlineCode> corresponds to <InlineCode>960px</InlineCode>.
+      </>
+    }
+  />
+  <PropsTableRow
+    name="height"
+    type={
+      <>
+        <InlineCode>xsmall</InlineCode>, <InlineCode>small</InlineCode>, <InlineCode>medium</InlineCode>,{' '}
+        <InlineCode>large</InlineCode>, <InlineCode>xlarge</InlineCode>, <InlineCode>auto</InlineCode>
+      </>
+    }
```

Comment:
```suggestion
    type="'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'auto'"
```

Reply:
@colebemis. Thanks. I've rolled back most of these changes to match your suggestions.

I do feel however, that it's less readable this way, what do you think? 

E.g.

Before:
<img width="310" alt="Screenshot 2022-01-18 at 20 38 03" src="https://user-images.githubusercontent.com/13340707/150015757-c7feb76b-e24c-46d2-8a03-8b2d3b602da0.png">

After:
<img width="271" alt="Screenshot 2022-01-18 at 20 43 32" src="https://user-images.githubusercontent.com/13340707/150015772-ff200b3c-1f98-44c5-81c8-dfc90f58d5f7.png">

Reply:
Agreed. Looks like we use newlines in `AnchoredOverlay` docs to solve this issue: https://primer.style/react/AnchoredOverlay#props

We could do that here too

Reply:
<img width="949" alt="CleanShot 2022-01-18 at 13 51 50@2x" src="https://user-images.githubusercontent.com/4608155/150024527-4368bb3b-d28f-46f5-bd45-5d59af61e727.png">

```
  <PropsTableRow
    name="side"
    type={`| 'inside-top' 
| 'inside-bottom' 
| 'inside-left' 
| 'inside-right' 
| 'inside-center' 
| 'outside-top' 
| 'outside-bottom' 
| 'outside-left' 
| 'outside-right'`}
    defaultValue="'outside-bottom'"
  />
```

Reply:
Much better. Thanks @colebemis
![Screenshot 2022-01-19 at 13 40 53](https://user-images.githubusercontent.com/13340707/150142262-f62443f6-f916-46d6-9c58-19bfc97c965e.png)